extensions/wikia/Forum/ForumHooksHelper.class.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+use Wikia\PageHeader\Button;
+
 class ForumHooksHelper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "ForumHooksHelper" has 31 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)